pkg/server/router/manifest.go

```diff
@@ -0,0 +1,183 @@
+	if !ok {
+		return nil, fmt.Errorf("could not create manifest router with service type: %s", s.Type())
+	}
+	return &ManifestRouter{
+		service: credService,
+	}, nil
+}
+
+type CreateManifestRequest struct {
+	Issuer string `json:"issuer" validate:"required"`
+	// A context is optional. If not present, we'll apply default, required context values.
+	Context                string                   `json:"@context"`
+	OutputDescriptors      []map[string]interface{} `json:"outputDescriptors" validate:"required"`
+	PresentationDefinition map[string]interface{}   `json:"presentationDefinition" validate:"required"`
+}
+
+func (c CreateManifestRequest) ToServiceRequest() manifest.CreateManifestRequest {
+	return manifest.CreateManifestRequest{
+		Issuer:                 c.Issuer,
+		Context:                c.Context,
+		OutputDescriptors:      c.OutputDescriptors,
+		PresentationDefinition: c.PresentationDefinition,
+	}
+}
+
+type CreateManifestResponse struct {
+	Manifest manifestsdk.CredentialManifest `json:"manifest"`
+}
+
+// CreateManifest godoc
+// @Summary      Create manifest
+// @Description  Create manifest
+// @Tags         ManifestAPI
+// @Accept       json
+// @Produce      json
+// @Param        request  body      CreateManifestRequest  true  "request body"
+// @Success      201      {object}  CreateManifestResponse
+// @Failure      400      {string}  string  "Bad request"
+// @Failure      500      {string}  string  "Internal server error"
+// @Router       /v1/manifests [put]
+func (cr ManifestRouter) CreateManifest(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	var request CreateManifestRequest
+	if err := framework.Decode(r, &request); err != nil {
+		errMsg := "invalid create manifest request"
+		logrus.WithError(err).Error(errMsg)
+		return framework.NewRequestError(errors.Wrap(err, errMsg), http.StatusBadRequest)
+	}
+
+	req := request.ToServiceRequest()
+	createManifestResponse, err := cr.service.CreateManifest(req)
+	if err != nil {
+		errMsg := "could not create manifest"
+		logrus.WithError(err).Error(errMsg)
+		return framework.NewRequestError(errors.Wrap(err, errMsg), http.StatusInternalServerError)
+	}
+
+	resp := CreateManifestResponse{Manifest: createManifestResponse.Manifest}
+
+	return framework.Respond(ctx, w, resp, http.StatusCreated)
+}
+
+type GetManifestResponse struct {
+	ID       string                         `json:"id"`
+	Manifest manifestsdk.CredentialManifest `json:"manifest"`
+}
+
+// GetManifest godoc
+// @Summary      Get manifest
+// @Description  Get manifest by id
+// @Tags         ManifestAPI
+// @Accept       json
+// @Produce      json
+// @Param        id   path      string  true  "ID"
+// @Success      200  {object}  GetManifestResponse
+// @Failure      400  {string}  string  "Bad request"
+// @Router       /v1/manifests/{id} [get]
+func (cr ManifestRouter) GetManifest(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
```

Comment:
nit: for schemas it's called `GetSchemaByID`, I think your name here is better, would you mind updating the schema to say `GetSchema` ?